agreement_serviceprofile/views/agreement.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Agreement Form View -->
+    <record id="agreement_fsm_order_form_view" model="ir.ui.view">
+        <field name="name">agreement.form.fsm.order.view</field>
+        <field name="model">agreement</field>
+        <field name="inherit_id" ref="agreement_legal.partner_agreement_form_view"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page name="serviceprofiles" string="Service Profiles">
+                    <field name="serviceprofile_ids">
+                        <tree>
+                            <field name="name"/>
+                        </tree>
```

Comment:
@osi-scampbell You can do something like this:

```suggestion
                        </tree>
                        </form>
                            <field name="name"/>
                            <field name="notes"/>
                        </field>
```

Reply:
> @osi-scampbell You can do something like this:
![image](https://user-images.githubusercontent.com/39602586/59226913-e8db2580-8b88-11e9-8599-649ebfc1c2a6.png)

I think that is a great idea, when you pull up the form view from that One2many field it has <p><br></p> in notes, I think it is because of the html widget. I added a placeholder and that seems to fix that issue.